frontend/public/components/start-guide.tsx

```diff
@@ -12,39 +13,46 @@ import { ProjectModel } from '../models';
 import { createProjectModal } from './modals/create-namespace-modal';
 
 export const OpenShiftGettingStarted = connect(createProjectMessageStateToProps)(
-  ({ canCreateProject = true, createProjectMessage }: OpenShiftGettingStartedProps) => (
-    <>
-      {canCreateProject ? (
+  ({ canCreateProject = true, createProjectMessage }: OpenShiftGettingStartedProps) => {
+    const { t } = useTranslation();
+    return (
+      <>
+        {canCreateProject ? (
+          <p>
+            {t(
+              'public~OpenShift helps you quickly develop, host, and scale applications. To get started, create a project for your application.',
+            )}
+          </p>
+        ) : (
+          <p>
+            {t(
+              'public~OpenShift helps you quickly develop, host, and scale applications. To get started, you&lsquo;ll need a project. Currently, you can&lsquo;t create or access any projects.',
+            )}
+            {!createProjectMessage &&
+              t('public~ You&lsquo;ll need to contact a cluster administrator for help.')}
+          </p>
+        )}
+        {createProjectMessage && (
+          <p className="co-pre-line">
+            <LinkifyExternal>{createProjectMessage}</LinkifyExternal>
+          </p>
+        )}
         <p>
-          OpenShift helps you quickly develop, host, and scale applications. To get started, create
-          a project for your application.
+          {t('public~To learn more, visit the OpenShift  ')}
```

Comment:
It would be better to do this using the component so we can preserve the context of the sentence for the translators. (Sometimes word order needs to change.)

Reply:
used trans component here